gb-cpu/src/processor/cpu/registers/area.rs

```diff
@@ -0,0 +1,41 @@
+#[derive(Debug)]
+pub enum Bits8 {
```

Comment:
where is:
- `A`
- `F` 
?

Reply:
are you're right for these 2 but they're accessed like any other register 8 bits

Reply:
+ they're decoded from normal opcode

Reply:
I'll add A there but i think we shoud use the bitfield crate that you used for F

Reply:
I'm not agains't using a bitfield for F **BUT** F is a valid register that's also used in opcode